src/data/automation.ts

```diff
@@ -219,8 +219,8 @@ export interface NumericStateCondition extends BaseCondition {
   condition: "numeric_state";
   entity_id: string;
   attribute?: string;
-  above?: number;
-  below?: number;
+  above?: string | number;
+  below?: string | number;
```

Comment:
Not directly related but actually wrong type: automation numeric_state can be a string as can be an entity_id.

See:
https://github.com/home-assistant/frontend/blob/dcb3accdb86f534277c16fd6d1c063f0a79c52dd/src/panels/config/automation/condition/types/ha-automation-condition-numeric_state.ts#L202-L208